packages/inter-protocol/test/vaultFactory/test-vaultLiquidation.js

```diff
@@ -63,6 +63,7 @@ const trace = makeTracer('TestST', false);
 
 const SECONDS_PER_DAY = SECONDS_PER_YEAR / 365n;
 const SECONDS_PER_WEEK = SECONDS_PER_DAY * 7n;
+const TEN_MINUTES = 10n * 60n;
```

Comment:
This doesn't say what the base unit it. When it does, I think multiplying by 10n at the site is appropriate.
```suggestion
const SECONDS_PER_MINUTE = 60n;
const SECONDS_PER_HOUR = 60n * SECONDS_PER_MINUTE;
```
The latter should make the `startFrequency` value more obvious.

Neither blocking.